katas/content/key_distribution/Common.qs

```diff
@@ -0,0 +1,31 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Random;
+
+    operation RandomArray(N : Int) : Bool[] {
+        ForEach(x => DrawRandomInt(0, 1) == 0, [0, size = N])
+    }
+
+    operation BasisToString(base : Bool) : String {
+        base ? "Hadamard" | "computational"
+    }
+
+    operation StateToString(base : Bool, bit : Bool) : String {
+        if base {  // ∣+⟩ / ∣-⟩
+            return bit ? "|-⟩" | "|+⟩";
+        } else {  // ∣0⟩ / ∣1⟩
+            return bit ? "|1⟩" | "|-⟩";
```

Comment:
```suggestion
            return bit ? "|1⟩" | "|0⟩";
```

Reply:
My suggestion assumes that the original return string when the bit is not set was unintended.